velox/connectors/hive/HiveConnector.cpp

```diff
@@ -416,6 +414,14 @@ void HiveDataSource::addSplit(std::shared_ptr<ConnectorSplit> split) {
         executor_,
         readerOpts_.loadQuantum(),
         readerOpts_.maxCoalesceDistance());
+  } else if (parallelLoadEnable_) {
```

Comment:
Can't we read the `HiveConfig` here and remove the variable `parallelLoadEnabled_`?

Reply:
Let me try to pass hive config or properties to HiveDataSource, but I think it will make a lot changes.

Reply:
You need to include the HiveConfig.h header to read `kParallelLoadEnabled` since the functions are static.

Reply:
I see you mean config is missing. This is fine.